app/views/consent_responses/edit_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :before_main do %>
+  <%= render AppBacklinkComponent.new(
+               href: session_patient_triage_path(@session, @patient),
+               name: "triage page",
+             ) %>
+<% end %>
+
+<h1 class="nhsuk-heading-l">Health questions</h1>
+
+<%= form_for @draft_consent_response,
```

Comment:
Just thinking ... for now this is a draft `consent_response` but this should with a non-draft `consent_response` object too, right? So at some point this will just be `@consent_response` I guess.